### PR TITLE
Fix: Do not use real time in tests

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

Two days ago, I had a unit test failure because a test was relying on real time instead of setting the system time.

In this PR: I looked at all the tests that were using real time and changed it.

# Changes

* Set time for NnsNeuronCard.spec.
* Set time and clarify dependencies of creation times with a variable in SnsVotingCard.spec.
* Set time in NnsProposalCard.spec
* Set time in sns-neuron.utils.spec

# Tests

Only test changes.

# Todos

- [x] Add entry to changelog (if necessary).

### CODE REVIEW

**Comment by lmuntaner** [comment]
@dskloetd please review

File: frontend/src/tests/lib/components/proposal-detail/VotingCard/SnsVotingCard.spec.ts [hunk 1]
```diff
@@ -63,13 +65,15 @@ describe("SnsVotingCard", () => {
...
+  const neuronCreatedAt = BigInt(nowInSeconds - SECONDS_IN_DAY * 2);
+  const proposalCreatedAt = BigInt(nowInSeconds - SECONDS_IN_DAY);
   const testProposal: SnsProposalData = {
     ...createSnsProposal({
       proposalId: 123n,
       status: SnsProposalDecisionStatus.PROPOSAL_DECISION_STATUS_OPEN,
       rewardStatus: SnsProposalRewardStatus.PROPOSAL_REWARD_STATUS_ACCEPT_VOTES,
       ballots: testBallots,
-      createdAt: BigInt(Math.round(Date.now() / 1000)),
+      createdAt: proposalCreatedAt,
```
Comment (dskloetd) [inline frontend/src/tests/lib/components/proposal-detail/VotingCard/SnsVotingCard.spec.ts:76]: Previously the proposal was created now, but now it's created yesterday.
Was this change necessary?

Comment (dskloetd) [reply frontend/src/tests/lib/components/proposal-detail/VotingCard/SnsVotingCard.spec.ts:76]: Thanks for clarifying.

**Review by dskloetd — APPROVED** [review]
